cvpysdk/subclient.py

```diff
@@ -1722,3 +1732,54 @@ def _set_subclient_content_(self, subclient_content):
             raise SDKException('Subclient', '102', '{} not given in content'.format(err))
 
         return content
+
+
+class CloudAppsSubclient(Subclient):
+    """Derived class from Subclient Base class, representing a CloudApps subclient,
+        and to perform operations on that subclient."""
+
+    def _get_subclient_content_(self, subclient_properties):
+        """Gets the appropriate content from the Subclient relevant to the user.
+
+            Args:
+                subclient_properties (dict)  --  dictionary contatining the properties of subclient
+
+            Returns:
+                list - list of content associated with the subclient
+        """
+        content = []
+
+        for account in subclient_properties['content']:
+            content_dict = {}
+            temp_account = account["cloudconnectorContent"]["includeAccounts"]
+            content_dict['SMTPAddress'] = temp_account["contentName"]
+            content_dict['displayName'] = temp_account["contentValue"]

```

Comment:
Rename 'displayName' to 'display_name' as per PEP8

Also please follow same design as Virtual Server Subclient